en/mkdocs.yml

```diff
@@ -64,6 +64,8 @@ nav:
                             - Generate REST API from SOAP Backend: learn/design-api/create-api/generate-rest-api-from-soap-backend.md
                       - Adding Custom Properties to APIs: learn/design-api/create-api/adding-custom-properties-to-apis.md
                       - Change the Thumbnail of an API : learn/design-api/create-api/change-api-thumbnail.md
+                - Test APIs:
```

Comment:
As mentoned above, can we move these docs under Create API?

Reply:
fixed